extensions/resource-deployment/src/services/tools/azdataTool.ts

```diff
@@ -58,13 +58,17 @@ export class AzdataTool extends ToolBase {
 		return true;
 	}
 
-	protected async getInstallationPath(): Promise<string | undefined> {
+	protected async getInstallationPath(): Promise<string[]> {
```

Comment:
Update this to getInstallationPaths now that we can return more than one path? Although shouldn't these go into installationSearchPaths instead? Not sure why we'd ever have more than one "installation path"

Reply:
we have more than one installationPath because we could be installing different versions of same tool from user configurable setting. And each version could potentially be installed in a different location.

Reply:
will change to getInstallationPaths.

Reply:
done.

Reply:
@ranasaria please push your commit. I am working on your branch now.